src/js/controller/controller.js

```diff
@@ -283,7 +283,7 @@ Object.assign(Controller.prototype, {
             }
 
             // Autostart immediately if we're not waiting for the player to become viewable first.
-            if (_model.get('autostart') === true) {
+            if (_model.get('autostart') === true && !_model.get('playOnViewable')) {
```

Comment:
would it make more sense to check for `!_model.get('playOnViewable')` before even calling `_checkAutoStart` ?

Reply:
@karimJWP They're both related to autostart, so imo it make sense to have this logic in a function called `_check*`.